config/apis.ts

```diff
@@ -40,16 +51,27 @@ const keyAuthAPIList: KeyAuthAPI[] = [
     key: 'claimsAttributes',
     name: 'Claims Attributes API',
   },
+  {
+    acl: 'hlr',
+    key: 'decisionReviews',
```

Comment:
why this key? on the frontend it's `decision_reviews`, and i would rather not force @keithosayande to make any more frontend changes. same for Loan Guaranty